www/gatsby-node.js

```diff
@@ -396,6 +408,83 @@ exports.onCreateNode = ({ node, actions, getNode, getNodes }) => {
     slug = `/showcase/${slugify(cleaned)}`
     createNodeField({ node, name: `slug`, value: slug })
   }
+  // Starter Showcase Pages
+  else if (node.internal.type === `StartersYaml` && node.repo) {
+    if (!process.env.GITHUB_TOKEN) {
+      console.log(
+        "You need to create and set a Github token to work with the starter showcase"
+      )
+      return
+    }
+    const [owner, slug] = node.repo.split(`/`).splice(-2, 2)
+
+    Promise.all([
```

Comment:
A potentially more readable way to do this would be something like this:

```js
const [pkgjson, githubData] = await Promise.all(/* ... */);
```

It unwraps that Promise and avoids the `.then` nesting. This is a preference, not an optimization, though, so feel free to ignore.